theme/templates/collection.liquid

```diff
@@ -1,18 +1,5 @@
-{% paginate collection.products by 2 %}
-  <h1>{{ collection.title }}</h1>
+<div class="Collection grid">
   {% for product in collection.products %}
-    <div>
-      <a href="{{ product.url | within: collection }}">{{ product.title }}</a>
-      {{ product.price | money }}
-      {% unless product.available %}<br><strong>sold out</strong>{% endunless %}
-      <a href="{{ product.url | within: collection }}">
-        <img src="{{ product.featured_image.src | img_url: 'large' }}" alt="{{ product.featured_image.alt | escape }}">
-      </a>
-    </div>
-  {% else %}
-    <p>no matches</p>
+    {% render 'product-grid-item', current_product: product %}
```

Comment:
I think this will probably need to be adjusted later. For now I'm just showing the products in the 3 col grid without a featured product.

Reply:
Does this comment still apply?

Reply:
I'm going to revert this back to the template and take care of it when I know more! Disregard :)